docs/docs/developer_guide/documentation.md

```diff
@@ -0,0 +1,68 @@
+# Mechaphlowers Documentation
+
+## Building documentation 
+
+### Serving documnentation
+
+From inside docs folder : 
+```{bash}
+cd ./docs
+mkdocs serve -a 0.0.0.0:8001
```

Comment:
Maybe this piece of documentation should be moved to the same file where we describe the other commands that are useful for mechaphlowers - that is, the README.md?